code/profiler/top.q

```diff
@@ -0,0 +1,16 @@
+if[`l64<>.z.o; -2"error: linux 64-bit is required";exit 0]
```

Comment:
convention is that `exit 0` means "success" and non-zero exit code is used for failure, probably best to make this non-zero